### ISSUE TITLE
Docker Tags Don't Exist

### ISSUE THREAD

**Issue by frankgreco** [kubernetes/heapster#1504]
The following Docker images/tags don't exist anymore

`gcr.io/google_containers/heapster-influxdb:v1.1.1`

`gcr.io/google_containers/heapster-grafana:v4.0.2`

**Comment by piosz** [comment]
Closing in favor of #1420. In the meantime please use:
```
gcr.io/google_containers/heapster-influxdb-amd64:v1.1.1
gcr.io/google_containers/heapster-grafana-amd64:v4.0.2
```
(or your architecture if you have something else than `amd64`).

Sorry for the confusion

**Comment by frankgreco** [comment]
@piosz I will use the above tags.

However, this is still an issue because the Kubernetes manifests are wrong. The manifests are not usable out of the box.

**Comment by piosz** [comment]
@frankgreco this should be fixed now by #1496.
